nucypher/cli/characters/alice.py

```diff
@@ -171,11 +185,28 @@ def alice(click_config,
             return actions.handle_missing_configuration_file(character_config_class=AliceConfiguration,
                                                              config_file=config_file)
 
+    if action == "destroy":
+        """Delete all configuration files from the disk"""
+        if dev:
+            message = "'nucypher alice destroy' cannot be used in --dev mode"
+            raise click.BadOptionUsage(option_name='--dev', message=message)
+        return actions.destroy_configuration(emitter, character_config=alice_config, force=force)
+
+    #
+    # Produce Alice
+    #
+
+    # TODO: OH MY.
+    client_password = None
+    if not alice_config.federated_only:
+        if (not hw_wallet or not dev) and not click_config.json_ipc:
```

Comment:
Is `hw_wallet` option used at all except here?

Reply:
This is the only reason the flag is used at the moment.  See related #1128